tests/integration/targets/ndo_fabric_span_session_source/tasks/main.yml

```diff
@@ -0,0 +1,1848 @@
+
+    - name: Update existing L3Out value in the SPAN Session source
+      cisco.mso.ndo_fabric_span_session_source:
+        <<: *mso_info
+        template: ansible_test_mon_pol
+        span_session_name: ansible_test_span_session
+        name: ansible_test_source_1
+        direction: outgoing
+        access_paths:
+          - access_path_type: port_channel
+            name: ansible_test_pc1
+            template: ansible_test_fabric_resource
+        filter_l3out:
+          vlan_id: 42
+          l3out:
+            name: ansible_test_l3out
+            template: ansible_test_l3out_template
+        state: present
+      register: update_filter_l3out_ansible_test_source_1
+
+    - name: Add EPG to the SPAN Session source again
+      cisco.mso.ndo_fabric_span_session_source:
+        <<: *mso_info
+        template: ansible_test_mon_pol
+        span_session_name: ansible_test_span_session
+        name: ansible_test_source_1
+        direction: outgoing
+        filter_epg:
+          uuid: "{{ update_filter_epg_ansible_test_source_1.current.epg }}"
+        state: present
+      register: add_filter_epg_ansible_test_source_1_again
+
+    - name: Add L3Out to the SPAN Session source again with UUID
+      cisco.mso.ndo_fabric_span_session_source:
+        <<: *mso_info
+        template: ansible_test_mon_pol
+        span_session_name: ansible_test_span_session
+        name: ansible_test_source_1
+        direction: outgoing
+        access_paths:
+          - access_path_type: port_channel
+            name: ansible_test_pc1
+            template: ansible_test_fabric_resource
+        filter_l3out:
+          uuid: "{{ add_l3out.current.uuid }}"
+          vlan_id: 42
+        state: present
+      register: add_filter_l3out_ansible_test_source_1_uuid
+
+    - name: Add L3Out to the SPAN Session source with APIC tenant
+      cisco.mso.ndo_fabric_span_session_source:
+        <<: *mso_info
+        template: ansible_test_mon_pol
+        span_session_name: ansible_test_span_session
+        name: ansible_test_source_1
+        direction: outgoing
+        access_paths:
+          - access_path_type: port
+            node: 101
+            interface: eth1/24
+        filter_l3out:
+          vlan_id: 24
+          l3out:
+            name: apic_l3out
+            tenant: apic_ansible_test_tenant
+        state: present
+      register: add_filter_l3out_ansible_test_source_1_tenant
+      ignore_errors: true
+
+    - name: Remove L3Out and access paths from the SPAN Session source
+      cisco.mso.ndo_fabric_span_session_source:
+        <<: *mso_info
+        template: ansible_test_mon_pol
+        span_session_name: ansible_test_span_session
+        name: ansible_test_source_1
+        direction: outgoing
+        access_paths: []
+        filter_l3out: {}
+        state: present
+      register: update_filter_l3out_ansible_test_source_2
+
+    - name: Assertion check for update SPAN Session source with different options
```

Comment:
This assesrtion is task is too big, it has to split based on add, remove, update according to the tasks